substrate/frame/src/lib.rs

```diff
@@ -202,10 +202,17 @@ pub mod prelude {
 	/// Dispatch types from `frame-support`, other fundamental traits
 	#[doc(no_inline)]
 	pub use frame_support::dispatch::{GetDispatchInfo, PostDispatchInfo};
-	pub use frame_support::traits::{
-		Contains, EitherOf, EstimateNextSessionRotation, Everything, IsSubType, MapSuccess,
-		NoOpPoll, OnRuntimeUpgrade, OneSessionHandler, RankedMembers, RankedMembersSwapHandler,
-		VariantCount, VariantCountOf,
+	pub use frame_support::{
+		defensive, defensive_assert, impl_ensure_origin_with_arg_ignoring_arg,
+		storage::KeyLenOf,
+		traits::{
+			BalanceStatus, Contains, Currency, Defensive, DefensiveOption, EitherOf, EnsureOrigin,
```

Comment:
`Currency` is kind of deprecated in favor of fungible no? It would make sense not to have it in the prelude then maybe. But it is also ok to me.